app/app.js

```diff
@@ -34,7 +34,7 @@ openSansObserver.check().then(() => {
 });
 
 // Import the CSS reset, which HtmlWebpackPlugin transfers to the build folder
-import '../node_modules/sanitize.css/dist/sanitize.min.css';
+import '../node_modules/sanitize.css/sanitize.css';
```

Comment:
We don't really want to do that though, we just want to add it to our stylesheet. If you want to compile it from source, I think you should add it manually, not from node_modules...

That problem was recently fixed, see: https://github.com/mxstbr/react-boilerplate/blob/v3.0.0/internals/webpack/webpack.base.babel.js#L26-L33